R/loopRuns.R

```diff
@@ -36,59 +36,66 @@ loopRuns <- function(mydir, user = NULL, colors = TRUE, sortbytime = TRUE) {
   if (file.exists("/p")) {
     coltitles <- c(paste0("Folder", paste(rep(" ", len - 6), collapse = "")),
       "Runtime    ", "inSlurm ", "RunType    ", "RunStatus        ", "Iter            ",
```

Comment:
shorten the column length a bit to increase the chance that everything fits in one line. The `Mif` column was always just too big, but now an even narrower column is possible by using `yes` and `no` instead of `TRUE` and `FALSE`